easybuild/easyconfigs/s/SQLite/SQLite-3.8.10.2-GCC-4.9.3-2.25.eb

```diff
@@ -0,0 +1,38 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'SQLite'
+version = '3.8.10.2'
+
+homepage = 'http://www.sqlite.org/'
+description = 'SQLite: SQL Database Engine in a C Library'
+
+toolchain = {'name': 'GCC', 'version': '4.9.3-2.25'}
+
+# eg. http://www.sqlite.org/2014/sqlite-autoconf-3080600.tar.gz
+source_urls = ['http://www.sqlite.org/2015/']
+version_str = '%(version_major)s' + ''.join('%02d' % int(x) for x in version.split('.')[1:])
+sources = ['sqlite-autoconf-%s.tar.gz' % version_str]
+
+dependencies = [
+    ('libreadline', '6.3', '', ('GCCcore', '4.9.3')),
+    ('Tcl', '8.6.4'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/sqlite3'],
+    'dirs': [],
```

Comment:
recent SQLite3 easyconfigs have a more extended `sanity_check_paths`, to also check header files and libraries, use that?